vividus-engine/src/main/java/org/vividus/batch/BatchConfiguration.java

```diff
@@ -167,19 +166,25 @@ private List<String> convertToList(String list)
 
     private static final class StoryExecutionConfiguration
     {
-        private boolean failFast;
+        private Boolean failFast;
+        private Duration executionTimeout;
 
-        private void setFailFast(boolean failFast)
+        private void setFailFast(Boolean failFast)
```

Comment:
why is it requred to change to boxed type?

Reply:
To Keep previous behaviour